src/graphql/projects/projectTL.graphql

```diff
@@ -0,0 +1,130 @@
+fragment ProjectTLParts on Project {
+  currentBalance
+  volume
+  trendingScore
+}
+
+query ProjectTL(
+  $id: ID!
+  $block0: Int
```

Comment:
yes this is gross, but it works well. using aliases lets us effectively make multiple unique queries in a single http req

here we want to get project data at 30 different historical block heights. unfortunately graphql doesn't allow for any logic or iterators, so we have to provide each block height as a named argument 🥲

Reply:
lgtm